Add graphic showcasing evolution of life

hendricius opened the pull request:
After reading a really good book by Ed Young on microorganism I figured I update the history of sourdough bread a bit more.

<img width="603" alt="image" src="https://github.com/hendricius/the-sourdough-framework/assets/816859/94f0ea88-4cf9-4b3b-b295-623c20a3d6d4">

cedounet commented:
I am not sure i understand the scaling on your timeline?  you normalize earth age to 1 year?  IMO it would be clearer if you put 20M years ago etc...  

I promise I will not be asking for a log scale 🤓

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
-Sourdough has been made since ancient times. The exact origins of fermented
+The story of sourdough bread begins in ancient oceans. These oceans were the
+birthplace of all Earth's life. To understand this timeline, envision Earth's
+history compressed into one year. On this scale, January 1st signifies Earth's
```
Not sure exactly how you do that but shall be upper script.

also insécable space between month and day.

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
+    marked at midnight. This visualization shows the pivotal steps
+    of life and sourdough on earth.}%
+\end{center}
+\end{figure}
```
Not sure this is clearer than showing millions of years… I see why you are trying to say but I need to think about it.

hendricius replied:
I got the idea from Ed Young's book "I contain multitudes". I kind of like it as it shows how recent all human innovation has been. Also shows how lonely the planet has been for some time. My plan is to add another one with 12000 BC till now once this PR is merged 😅

cedounet replied:
Have you tried to also have non scaled dates maybe in a different colour?

I would also add some key dates like Pangea (you already have dinosaurs extinction).  Just an idea.

hendricius replied:
Pangea is a funny idea 👍 . Just not sure I can fit more into that one chart 😅

hendricius replied:
Also - TLDR - it was that recent the continents formed 😅

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
+\end{center}
+\end{figure}
+
+The first arrivals, bacteria or archaea, remains debated. For three
```
What do you mean by debated ? Who arrived first ?

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
+years later, or our timeline's 2 days, humans appeared.
+
+A few hours later after the arrival of humans, a more subtle culinary
+revolution was unfolding. By 12,000 BC, just 5 seconds before our metaphorical
```
Use Siunit package here there is an example further down the text \num{12000}~BC

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
+an eye later, or 4 seconds in our time compression, Pasteur's groundbreaking work
+with yeasts set the stage for modern bread-making. From the moment this book
+began to take shape to your current reading, only milliseconds have ticked by.
+~\cite{Yong_2017}
```
Put final dot after the citation.

hendricius commented:
@cedounet thanks for the Feedback! I fixed most of the issues 👍 . As a basis to calculate I used: "(4.54 billion years / 12 months = 0.3785 billion years/month". I also added it as a comment in the code 😎

cedounet commented:
You might want to reword your dates as well to match American style https://editorsmanual.com/articles/dates-american-vs-british/

this is pretty much what I have in the Chicago manual of style… not ordinals.

if we ever want to translate then we should use datetime2 package.  I would use it anyway as I would get the date format wrong without looking on the inter webs.

hendricius commented:
> You might want to reword your dates as well to match American style https://editorsmanual.com/articles/dates-american-vs-british/
> 
> this is pretty much what I have in the Chicago manual of style… not ordinals.
> 
> if we ever want to translate then we should use datetime2 package. I would use it anyway as I would get the date format wrong without looking on the inter webs.

Sweet! Thanks for the tip. I updated - hope I didn't miss something. `datetime2` sounds like a good idea too. Let me see if I can add this with the next PR for the sourdough human timeline.

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -28,7 +78,7 @@ \chapter{The history of sourdough}%
 the dough anyway and was rewarded with a much
 lighter, softer, better tasting bread dough. From that day
-on she continued to make bread this way.
+on she continued to make bread this way.~\cite{egyptian+bread}
```
Punctuation vs quote

hendricius commented:
Thanks! I fixed the issues. Also added pangea and squeezed it in. Awesome idea. Can you check one more time please? Thank you!

cedounet commented on book/history/sourdough-history.tex:
```diff
@@ -9,11 +9,66 @@ \chapter{The history of sourdough}%
-sourdough breads has been excavated in Switzerland.
-However, based on recent research, some scientists speculate that sourdough
-bread had already been made in \num{12000}~BC in ancient Jordan~\cite{jordan+bread}.
+sourdough breads has been excavated in Switzerland.~\cite{switzerland+bread}
```
Citation before the dot

cedounet reviewed (commented):
Maybe run make check on it.  I could only spot one citation mistake.

text ok time chart could be slanted for better readability maybe?

I would add the scaling bottom left of the graph… 1 quarter represents 1 billion year about.

hendricius commented:
Great idea 👍. Also thanks for the hint on make check.

On Mon 6. Nov 2023 at 10:20, cedounet ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Maybe run make check on it
>
> I would add the scaling bottom left of the graph… 1 quarter represents 1
> billion year about.
> ------------------------------
>
> In book/history/sourdough-history.tex
> <https://github.com/hendricius/the-sourdough-framework/pull/270#discussion_r1382648197>
> :
>
> >  bread are, however, unknown. One of the most ancient preserved
> -sourdough breads has been excavated in Switzerland.
> -However, based on recent research, some scientists speculate that sourdough
> -bread had already been made in \num{12000}~BC in ancient Jordan~\cite{jordan+bread}.
> +sourdough breads has been excavated in Switzerland.~\cite{switzerland+bread}
>
> Citation before the dot
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hendricius/the-sourdough-framework/pull/270#pullrequestreview-1714057643>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGHNW26Y2GMWE6QKGBWU6DYC77KHAVCNFSM6AAAAAA6YLRFPKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTOMJUGA2TONRUGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

cedounet commented:
I really like it now.  Have you tried to use colours?

hendricius commented:
Great idea 🙏🏻

On Mon 6. Nov 2023 at 22:59, cedounet ***@***.***> wrote:

> I really like it now. Have you tried to use colours?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hendricius/the-sourdough-framework/pull/270#issuecomment-1794463774>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGHNW6Q6YNPJSKQRVX2KQDYDCYG3AVCNFSM6AAAAAA6YLRFPKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJUGQ3DGNZXGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

hendricius commented:
@cedounet what do you think? I also didn't want to overdo the colors.

<img width="896" alt="image" src="https://github.com/hendricius/the-sourdough-framework/assets/816859/4b30490f-5249-4f93-9917-39e70d2a9f37">

cedounet commented:
Yeah it's a matter of balance... LGTM, we can always fine-tune later anyway.
